go.mod

```diff
@@ -33,8 +33,8 @@ require (
 	//
 	// when updating k8s.io/kubernetes, make sure to update the replace section too
 	//
-	k8s.io/api v0.31.2
-	k8s.io/apimachinery v0.31.2
+	k8s.io/api v0.31.3
+	k8s.io/apimachinery v0.31.3
```

Comment:
@nixpanic same change is required in replace section as well.

Reply:
Done!